packages/app-admin-ui/client/pages/Item/index.js

```diff
@@ -373,8 +378,9 @@ const ItemNotFound = ({ adminPath, errorMessage, list }) => (
   </PageError>
 );
 
-const ItemPage = ({ list, itemId, adminPath, getListByKey, toastManager }) => {
+const ItemPage = ({ list, itemId, adminPath, getListByKey }) => {
   const itemQuery = list.getItemQuery(itemId);
+  const { addToast } = useToasts();
```

Comment:
Might as well pass on all the functionality here? It's being deconstructed later on [line 459](https://github.com/keystonejs/keystone-5/pull/1544/files#diff-18fd6ba6fe6ea61ebfb3250f37792127R459).

```suggestion
  const toastManager = useToasts();
```